blockchain/blocks/src/tipset.rs

```diff
@@ -202,6 +202,24 @@ impl Tipset {
     pub fn tip_epoch(&self) -> &ChainEpoch {
         self.blocks[0].epoch()
     }
+    /// Returns true if tipsets are equal
+    pub fn equals(&self, ots: Tipset) -> bool {
+        if self.blocks().is_empty() && ots.blocks().is_empty() {
+            return true;
+        }
```

Comment:
A valid tipset should not have empty blocks, so weird to me that this check exists